model_analyzer/record/metrics_manager.py

```diff
@@ -347,35 +347,74 @@ def _create_model_variant(
         repository and fills directory with config
         """
 
-        variant_name = variant_config.variant_name
         if self._config.triton_launch_mode != "remote":
-            model_repository = self._config.model_repository
+            self._create_non_remote_mode_model_variant(
+                original_name, variant_config, ignore_first_config_variant
```

Comment:
What does ignore_first_config_variant do?

what is the difference in the code between remote vs non-remote? As far as I can tell the only difference is the use of ignore_first_config_variant. Is that true?

Reply:
See my above comment. It's not relevant for remote mode because there is no actual model to write into the directory. The difference btw. remote and non-remote is that non-remote will have the `config.pbtxt` along with the model; remote mode only has the `config.pbtxt` (since the model is on a remote machine and MA doesn't have access to it).